specifyweb/frontend/js_src/lib/components/Forms/BaseResourceView.tsx

```diff
@@ -139,7 +144,7 @@ export function useResourceView<SCHEMA extends AnySchema>({
         </>
       ),
     title,
-    formElement: form,
+    formElement: formRef.current,
```

Comment:
Could you please add a comment here regarding the following?
Remember how we talked about never using a value of a ref during render as updates to ref don't trigger re-render, thus this is likely to cause a bug.
Well, this place is an exception because we update the ref on each render (line 127), so the value is always up to date.
Please summarise/paraphrase the above information and put it into a comment